tests/test_tecan_od_analyzer.py

```diff
@@ -246,10 +242,17 @@ def test_interpolation(self):
 		result = interpolation("od_measurements.tsv", df_annotations, mean_df_bs)
 		expected_result = pd.read_excel("expected_interpolation_results.xlsx")
 		result = pd.read_excel("interpolation_results.xlsx")
+		
 		#output type
 		self.assertIsInstance(result, pd.DataFrame)
-		#Comparison to expected output
-		#self.assertTrue(expected_result["Estimation"].tolist()-0.01<=result["Estimation"].tolist()<= expected_result["Estimation"].tolist()+0.01)
+
+		expected_result = expected_result["Estimation"].tolist()
+		result = result["Estimation"].tolist()
+
+		for i in range(len(result)):
+			
+			#Comparison to expected output
+			self.assertTrue(result[i]-0.5<= expected_result[i]<= result[i]+0.5)
```

Comment:
The tolerance used seems to be quite a bit higher than what would be normally used for floating point number comparison.  Is there some sort of stochasticity in the calculations?